declaration/middleware.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+from django.conf import settings
+from django.core.urlresolvers import reverse_lazy
+from django.http import HttpResponseRedirect
+from django.template.defaultfilters import urlencode
+
+from .views import DECLARATION_COOKIE_KWARGS
+
+
+PROTECTED_URL_PATTERNS = settings.PROTECTED_URL_PATTERNS
+
+
+class DeclarationMiddleware(object):
```

Comment:
That wont happen using the import statement above. But if it isnt needed for now thats fine.